src/vendor/wordpress/_wordpress.scss

```diff
@@ -55,6 +55,55 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: File download button
+ * Applies our pattern library styles to gutenberg file download blocks.
+ */
+
+.wp-block-file {
+  /**
+   * 1. Prevents stacked file buttons from collapsing on each other. 
+   */
+
+  + .wp-block-file {
+    margin-top: $wp-button-gap; /* 1 */
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying buttons opacity on hover.
+   */
+
+  .wp-block-file__button,
+  * {
```

Comment:
Can you add a comment somewhere explaining why you're targeting the class AND the universal selector? Earlier you had this comment, which was helpful to prevent someone else coming in later and thinking they can optimize your code by removing the class:

```css
/*
 * If I don't include `wp-block-file__button` here, this selector is not 
 * specific enough to enforce the desired styles, as rules from the 
 * `@wordpress/block-library` override  various properties.
 */
```

Reply:
I don't think you need to repeat the comment in each selector — maybe add a footnote comment to each that references the earlier footnote?

Reply:
Thanks for your feedback @spaceninja. 

> maybe add a **footnote comment** to each that references the earlier footnote?

Forgive my ignorance here, but I wasn't actually sure how to implement a "footnote comment" _or_ a numerical comment, in the same way we do on prosperities on a classname. 

So...this is what I ended up with: 

```css
/**
   * The following three selectors target a class name _and_ a universal 
   * selector (*). This is intentional because the Gutenberg file block can
   * generate two different elements, a file-link and a file-button and we
   * want to style both of them with the same visual appearance and
   * characteristics. We could easily have chosen `:not(.wp-block-file__button)`
   * but opted for less characters.
   * 
   * 1. Prevents default WP styling from modifying buttons opacity on hover.
   */

  .wp-block-file__button,
  * {
    @include button.default;
    opacity: inherit !important; /* 1 */
  }

  ...
```